# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/contextualidentity/test/browser/browser_saveLink.js
# Commit: a45df1876e37
# Full Hash: a45df1876e375459648251484c7d5552d2bdae6d
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-01 16:21:08
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 1a85cc92d2fb -r a45df1876e37 browser/components/contextualidentity/test/browser/browser_saveLink.js
--- a/browser/components/contextualidentity/test/browser/browser_saveLink.js	Mon Jun 01 10:29:06 2020 +0000
+++ b/browser/components/contextualidentity/test/browser/browser_saveLink.js	Mon Jun 01 10:29:34 2020 +0000
@@ -13,7 +13,13 @@
 
   // make sure userContext is enabled.
   await SpecialPowers.pushPrefEnv({
-    set: [["privacy.userContext.enabled", true]],
+    set: [
+      ["privacy.userContext.enabled", true],
+      // This test does a redirect from https to http and it checks the
+      // cookies. This is incompatible with the cookie sameSite schemeful
+      // feature and we need to disable it.
+      ["network.cookie.sameSite.schemeful", false],
+    ],
   });
 });
 